shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -313,11 +322,7 @@ TizenGeometry TizenViewElementary::GetGeometry() {
 }
 
 void TizenViewElementary::SetGeometry(TizenGeometry geometry) {
-  EvasObjectResize(image_, geometry.width, geometry.height);
-  evas_object_move(image_, geometry.left, geometry.top);
-  evas_object_image_size_set(image_, geometry.width, geometry.height);
-
-  EvasObjectResize(container_, geometry.width, geometry.height);
+  EvasObjectResizeWithMinMaxHint(container_, geometry.width, geometry.height);
```

Comment:
`SetGeometry` is based on `container_` but `GetGeometry` is based on `image_`. Did you intend this?

Reply:
I fixed it. In addition, most of the operations have been modified to be based on the container handle.